tools/db/README.md

```diff
@@ -138,3 +138,12 @@ Using that command will result in a replication for every database that matches
 ### Replaying a snapshot
 
 To replay a snapshot, swap `--sourceDbUrl` and `--targetDbUrl` and call the script with the `replay` command. That command takes only 1 parameter: `--dbPrefix` to determine which backup to play back. Matching databases will be replicated back to the target database with the `backup_${TIMESTAMP_IN_SECONDS}_` removed, so they'd look just like the original database.
+
+## Database migration to new schema
```

Comment:
@csantanapr, @rabbah, what do you think about including the action migration doc in this README?

Reply:
Yes add section in the this README also since the script is in the same folder.
It will be one place for db things ♻️